### PR TITLE
Support for container status

### PR DESCRIPTION
**Pull request by mrunalp**
Add support for container status. 

@runcom PTAL.

### CODE REVIEW

File: server/runtime.go [hunk 1]
```diff
@@ -627,7 +627,31 @@ func (s *Server) ListContainers(context.Context, *pb.ListContainersRequest) (*pb
...
 
 // ContainerStatus returns status of the container.
-func (s *Server) ContainerStatus(context.Context, *pb.ContainerStatusRequest) (*pb.ContainerStatusResponse, error) {
+func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusRequest) (*pb.ContainerStatusResponse, error) {
+	containerName := req.ContainerId
+
+	if *containerName == "" {
+		return nil, fmt.Errorf("container ID should not be empty")
+	}
+	c := s.state.containers[*containerName]
```
Comment (runcom) [inline server/runtime.go]: Should this be guarded somehow?

**Comment by runcom** [comment]
LGTM (just a nit, feel free to merge as is we'll refactor later if needed)

File: server/runtime.go (hunk 1 above)
Comment (mrunalp) [inline server/runtime.go]: Yeah, we'll need that. I have that in my TODO. Thanks.

Comment (runcom) [inline server/runtime.go]: :+1:

**Pull request merged**